spec/features/auth_spec.rb

```diff
@@ -2,18 +2,29 @@
 require 'capybara/rails'
 
 feature 'Auth' do
+#
+#   scenario 'Users can login and out' do
+#     create_user email: "user@example.com"
+#
+#     visit root_path
+#     within(".signin-form") { click_on "Login" }
+#     expect(page).to have_content("Username / password is invalid")
+#
+#     fill_in "Email", with: "user@example.com"
+#     fill_in "Password", with: "password"
+#     within(".signin-form") { click_on "Login" }
+#     expect(page).to have_content("user@example.com")
+#   end
```

Comment:
Checking in commented code is not always optimal. Consider when you're on a team that some other developer will come along and say "why is this all commented out? should it be?"